game/ato/flightplans/airlift.py

```diff
@@ -114,24 +133,25 @@ def layout(self) -> AirliftLayout:
             # Normal Landing Waypoint
             arrival = builder.land(self.flight.arrival)
         else:
-            # The AI Needs another Stopover point to actually fly back to the original
+            # The AI Needs another landing&refuel point to actually fly back to the original
```

Comment:
What original base? This is the arrival waypoint. It's the final destination. They should not go back anywhere.

Reply:
just a small typo. Why we need this is described above

Reply:
Described where? One of us has clearly misunderstood this code, because you think it's fine but I see a bug.

Reply:
I have to verify but i am quite sure that it is required to set an additional LandRefuel task waypoint AFTER it was used already once in the flightplan (in this case as the drop off airbase). If you do not use that type for the destination and just go with the landing task it will not land where you want. But to be sure i have to check.
Maybe ED fixed that stupid bug somehow.. It was one of the stupid DCS bugs i came across when testing airassault and airlift. 

Ill comment when i found some time watching AI doing super stupid stuff

Reply:
Oh, now I understand the comment. That's... bizarre. I haven't seen that bug. I can verify later.

Reply:
[Here's a track file](https://github.com/dcs-liberation/dcs_liberation/files/9947052/landing.bug.zip). Both flights take off, landrefuar at an intermediate airfield, then proceed to the final destination (the arrival airfield). The F-16 uses a regular landing action at the destination, and the F-5 uses a landrefuar. The F-16 lands and stays put. The F-5 lands, takes off again, then lands again. landrefuar implies another takeoff.

Unless I've still misunderstood, I think the track proves that this `else` branch is a bug and should be deleted (along with the supporting code)?

Reply:
Oh wonderful that you did the testing already. Then ED seems to have fixed that mess. so i will remove the code ofc :)